### PR TITLE
Update database guides

### PR DESCRIPTION
**Pull request by s1ntaxe770r**
This PR adds:

guides for creating databases using the CLI and Terraform

### CODE REVIEW

**Comment by s1ntaxe770r** [comment]
cc @civo-claire  @kai

**Review by kaihoffman — CHANGES_REQUESTED** [review]

File: content/docs/database/getting-started.md [hunk 1]
```diff
@@ -92,3 +100,232 @@ There are several reasons why one might add additional nodes to a database:
...
+civo_database.mysql_db: Still creating... [3m10s elapsed]
+civo_database.mysql_db: Still creating... [3m20s elapsed]
+civo_database.mysql_db: Still creating... [3m30s elapsed]
+civo_database.mysql_db: Creation complete after 3m40s [id=97de98fb-62dd-4fd1-a147-82a9c8f8804d]
+
+Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
+```
+When the creation completes, refresh your [Civo dashboard](https://dashboard.civo.com/databases) and you will see there's a new database that has been created. Click it to see more details. It will look something like this:
+
+![Database created from Terraform appears on the dashboard](images/mysql-terraform-screen.png)
```
[the remark this answers is not in the document]
Comment (s1ntaxe770r) [reply content/docs/database/getting-started.md:323]: hey @kaihoffman  are you referring to the reference name `mysql_db` or the name of the database `mysql-tf`

[the remark this answers is not in the document]
Comment (kaihoffman) [reply content/docs/database/getting-started.md:323]: Yeah, I was confused, sorry! This looks OK now.

File: content/docs/database/getting-started.md [hunk 2]
```diff
@@ -92,3 +100,232 @@ There are several reasons why one might add additional nodes to a database:
...
+civo db credential civo-db
+
+      ID : 02d46ed0-f1eb-437c-9751-bbf9c1f32f91
+    Name : civo-db
+    Host : 74.220.17.158:0
+    Username : civo
+    Password : <password>
+```
+
+### Veiwing Database information on Civo CLI
```
Comment (kaihoffman) [inline content/docs/database/getting-started.md:143]: Typo. Should be "Viewing".

**Review by kaihoffman — CHANGES_REQUESTED** [review]

File: content/docs/database/images/mysql-terraform-screen.png
Comment (civo-claire) [inline content/docs/database/images/mysql-terraform-screen.png:1]: Could you replace with an image not in dark mode please? Thanks!

**Review by kaihoffman — APPROVED** [review]

**Pull request merged**